api/src/wfl/api/routes.clj

```diff
@@ -155,13 +157,13 @@
        ;; :exception pretty/exception
      :data {:coercion   reitit.coercion.spec/coercion
             :muuntaja   muuntaja-core/instance
-            :middleware [;; query-params & form-params
+            :middleware [exception-middleware
```

Comment:
Moving middleware makes me nervous but moving it up in the response chain makes sense.